era-compiler-solidity/tests/common.rs

```diff
@@ -2,7 +2,18 @@
 //! The Solidity compiler unit tests.
 //!
 
+#![allow(dead_code)]
```

Comment:
Hey,

There was a reason to put all this code to `mod.rs`:
https://doc.rust-lang.org/book/ch11-03-test-organization.html#:~:text=Having%20common%20appear,looks%20like%20this%3A

it's a convention. for Rust integration tests. Do you want to change it to `common.rs`?

Reply:
Hey, thanks for the link. I didn't know it mattered in this particular case. Changed back now!